packages/vue/src/components/organisms/SfFooter/SfFooter.js

```diff
@@ -0,0 +1,56 @@
+import Vue from "vue";
+import SfFooterColumn from "./_internal/SfFooterColumn.vue";
+
+Vue.component("SfFooterColumn", SfFooterColumn);
+
+export default {
+  name: "SfFooter",
+  props: {
```

Comment:
Shouldn't footer be purely presentational component without any logic? What is the point of this behavior?